pages/register.js

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from "react";
+import Link from "next/link";
+
+function SignIn() {
+    const [name, setName] = useState();
+    const [email, setEmail] = useState();
+    const [password, setPassword] = useState();
+    const [confirmPassword, setConfirmPassword] = useState();
+    const [passError, setPassError] = useState(false);
+
+    useEffect(() => {
+        validatePassword(password, confirmPassword);
+    }, [password, confirmPassword]);
+
+    function validatePassword(pass, confrimPass) {
+        let isValid = confrimPass === pass;
+        if (!isValid) {
+            setPassError(true);
+        }
+    }
+    async function handleSubmit(e) {
+        e.preventDefault();
+        let userData = {
+            name,
+            email,
+            password,
+        };
+
+        // Make call to backend to create user
+        const currentUrl = window.location.origin;
+        console.log(currentUrl)
+        const res = await fetch(`${currentUrl}/api/user/create`, {
+            method: "POST",
+            body: JSON.stringify(userData),
+            headers: {
+                "Content-Type": "application/json",
+            },
+        });
+
+        if (res.ok) {
+            const data = await res.json();
+
+            // registration success
+        } else {
+            //registration faled
+        }
+    }
+    return (
```

Comment:
formatting

Reply:
added plugin in vs code and run control+s in all files. Let me know if you want me change any settings in vs code configuration file?